simpleLooping/removeSpaceAtEnd.py

```diff
@@ -17,3 +17,11 @@
 #   removeSpaceAtEnd("")                  # return ""
 #   removeSpaceAtEnd("Hello  World!")     # return "Hello  World!"
 #   removeSpaceAtEnd("  Hello  World!  ") # return "  Hello  World!"
+from typing import List
+
+def removeSpaceAtEnd(s: str) -> str:
+    i = len(s)- 1
+    while i >= 0 and s[i] == " ":
```

Comment:
Coding practice not satisfy. Single character should use single quote ‘ ‘, instead of “ “.

Reply:
ok